### ISSUE TITLE
[Feature request] Manage -> Media

### ISSUE THREAD

**Issue by Darkhost1999** [RPCS3/rpcs3#13621]
### Quick summary

We have a manage -> screenshots and I have a suggestion.

### Details

What if we changed manage -> screenshots to manage -> media (mms).

We can then manage inside that 1 area rpcs3/ screenshots (like we do), rpcs3/recordings, and even rpcs3/sounds or music or something.

The idea is that vsh manages the dev_hdd0 path. We just don't want to mix ps3 path and rpcs3 paths for mms content. So I'm trying to think of a way we can just manage mms content in 1 location. And wanted to share the idea as we make a music, video, and picture viewer in the pre-existing manage screenshots utility.

**Comment by Megamouse** [comment]
Just casually dropping here that I optimized the screenshot window scroll and resize in #13756.
It was constantly reloading the entire screenshots during any sort of layout change.

**Comment by digant73** [comment]
is this FT satisfied by current build?

**Comment by Darkhost1999** [comment]
Nope, as you can see the UI/UX is split into separate locations. The goal of this ticket is merely to track progress of each and vocalize the desire to refactor then merge the 3 separate media management tools into 1 localized hub. It's more or less an actionable ticker for myself, Megamouse, or any other aspiring fullstack dev/established fullstack dev to take interest in and recognition of a multi-step progression starting with each feature fully functioning independently prior to migration and refactorization.
This ticket is longterm and may survive until days of considering beta as everything starts being cleaned up and polished into a unified system. It keeps VSH for the PS3 filesystems and RPCS3 separated this ticket prioritizing only RPCS3 media filesystems.

Potential tags - discussion - longterm
